src/renderer/assets/scss/main.scss

```diff
@@ -3,6 +3,15 @@ html {
   user-select: none;
 }
 
+.page-enter-active,
+.page-leave-active {
+  transition: opacity 0.1s;
+}
+.page-enter,
+.page-leave-to {
+  opacity: 0;
+}
```

Comment:
You added the classes, but I don't think you're using them anywhere. Am I wrong?

Reply:
They're used when pages are changed. For example, when going from home to settings

Reply:
Ah, it's a nuxt feature. Nice!